cpp/Platform.Delegates/Platform.Delegates.Delegate.h

```diff
@@ -87,27 +165,26 @@ namespace Platform::Delegates
 
         static bool AreBoundFounctionsEqual(const DelegateFunctionType &left, const DelegateFunctionType &right)
         {
-            const size_t size = sizeof(DelegateFunctionType);
+            constexpr size_t size = sizeof(DelegateFunctionType);
             std::byte leftArray[size] = { {(std::byte)0} };
             std::byte rightArray[size] = { {(std::byte)0} };
             new (&leftArray) DelegateFunctionType(left);
             new (&rightArray) DelegateFunctionType(right);
             //PrintBytes(leftArray, rightArray, size);
-            ApplyHack(leftArray, rightArray, size);
+            ApplyHack<size>(leftArray, rightArray);
             return std::equal(std::begin(leftArray), std::end(leftArray), std::begin(rightArray));
         }
 
-        // By resetting certain values we are able to compare functions correctly
-        // When values are reset it has the same effect as when these values are ignored
-        static void ApplyHack(std::byte *leftArray, std::byte *rightArray, const size_t size)
+        template<size_t size>
```

Comment:
Looks like the comment to the method disappeared. Is it true?

Reply:
It looks like I accidentally deleted the comments, I will return them in the next edits)

Reply:
done in https://github.com/linksplatform/Delegates/pull/10/commits/bb8fbfd0d2fdd42dc49b244a42c590075b102254